tests/layer_tests/pytorch_tests/test_div.py

```diff
@@ -27,11 +27,11 @@ def forward(self, input_tensor, other_tensor):
         return aten_div(rounding_mode), ref_net, "aten::div"
 
     @pytest.mark.parametrize(("input_array", "other_array"), [
-        [np.random.rand(5, 5), np.random.rand(1)],
-        [np.random.rand(5, 5, 1), np.random.rand(1)],
-        [np.random.rand(1, 1, 5, 5), np.random.rand(1)],
-        [np.random.rand(5, 5, 1), np.random.rand(5, 1)],
-        [np.random.rand(5, 5), np.random.rand(5, 5)],
+        [np.random.uniform(low=0.1, high=1.0, size=(5, 5)), np.random.rand(1)],
```

Comment:
I think you changed the wrong array, you should have changed divisor

Reply:
Oh yeah you are right. Fixed.